### DISCUSSION TITLE
Support for Wifi Connectivity

### DISCUSSION THREAD

**Discussion by Jadcock92** [fl4p/batmon-ha#262] [Ideas]
I might be in the wrong area, but I have a Daly 100A BMS with a Wifi+BLE adapter. My battery is out in my shed and my home assistant host is in my house. The BMS can reach the wifi but not reach the bluetooth. I noticed the MAC address of the BMS is the same on BLE and Wifi.

Is there an easy way to have this plugin scan on the LAN network for a BMS? Or does a plugin with that function already exist somewhere. 

I'm new to HA but have a intermediate understanding of networking, coding, and electronics.
Thanks for any input!

**Comment by garylankford8** [comment]


**Reply by Karatzie** [reply]
Hi @garylankford8 .
The Bluetooth Proxy of ESPHome does not work batmon-ha.

BR
Andreas
